db/contacts.json

```diff
@@ -0,0 +1,132 @@
+{
+	"contacts":[
```

Comment:
This should be an object of objects with keys for each contact. (Just like it will be when you pull it back from Firebase).